StripeIdentity/StripeIdentity/Source/NativeComponents/ViewControllers/PhoneOtpViewController.swift

```diff
@@ -100,8 +100,7 @@ class PhoneOtpViewController: IdentityFlowViewController {
         fatalError("init(coder:) has not been implemented")
     }
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    override func viewWillAppear(_ animated: Bool) {
         updateUI()
```

Comment:
just FYI this will be called everytime the view appears, say you leave the app come back to this screen this code will run, are you sure you need this?

Reply:
ah thanks for catching this, we dont' need it everytime. Updated to `viewDidAppear` instead